### PR TITLE
events: Allow sending logs in batches

### PR DESCRIPTION
**Pull request by zeenix**
As we intended.

### CODE REVIEW

**Review by DmitrySamoylov — APPROVED** [review]

**Review by DmitrySamoylov — APPROVED** [review]

File: types/src/lib.rs [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #[cfg(test)]
 mod tests {
```
Comment (jplatte) [inline types/src/lib.rs:2]: I think we can just remove the entire module.

**Comment by zeenix** [comment]
Actually, I think we should compress the logs as they could be large, given how function node works on per-frame bases. For that reason it'd make sense to just concatenate them with a `\n` separator and then store in the even as bzip2 compressed text. WDYT @DmitrySamoylov @jplatte?

**Comment by jplatte** [comment]
I don't think that's a good idea. We want these logs to be shown by Console, and web browsers don't ship any (de)compression APIs AFAICT, so we would have to include more code in our JS bundle to do the decompression. It seems like there are be some protocol-level compression oppurtunities for both TLS and TLS+WebSockets MQTT transports though that we could maybe use.

**Comment by zeenix** [comment]
> Actually, I think we should compress the logs as they could be large, given how function node works on per-frame bases. For that reason it'd make sense to just concatenate them with a `\n` separator and then store in the even as bzip2 compressed text. WDYT @DmitrySamoylov @jplatte?

or alternatively, runner can also look at the size of the log (and not only timeout) to decide when it's time to send the logs.

File: events/src/deployment.rs [hunk 2]
```diff
@@ -40,3 +40,34 @@ pub struct NodeLog {
...
 }
+
+#[derive(Serialize, Deserialize, Type, Debug, Clone)]
+pub struct NodeLogs {
+    pub logs: Vec<NodeLog>,
+}
+
+impl NodeLogs {
+    pub fn bytes_len(&self) -> usize {
+        self.logs.iter().fold(0, |len, log| len + log.msg.len())
```
Comment (jplatte) [inline events/src/deployment.rs:51]: Nit: Easier to read as:

```suggestion
        self.logs.iter().map(|log| log.len()).sum()
```

**Review by jplatte — APPROVED** [review]

**Comment by zeenix** [comment]
rebasing..

**Pull request merged**